### ISSUE TITLE
post-npm-usage-instructions-comment references wrong commit SHA

### ISSUE THREAD

**Issue by taras** [thefrontside/actions#10]
It looks like we're detecting incorrect SHA. What's up with that?

<img width="1054" alt="Screen Shot 2019-10-04 at 4 19 29 PM" src="https://user-images.githubusercontent.com/74687/66237435-0cd7ef00-e6c3-11e9-875e-04b76d77c77a.png">

**Comment by minkimcello** [comment]
@taras Good catch but this is also intentional. Github has its own internal commit SHA while it's doing its thing and @cowboyd said we should use the actual SHA instead of going back to the repository to hunt down the SHA we want to see.

**Comment by cowboyd** [comment]
Also, this will be made obsolete by the change to use a branch tag?

**Comment by minkimcello** [comment]
@cowboyd Do we still want to show both options or just one of the two? Because installing with tag will "expire" once the branch is deleted. Maybe that's not an issue?

Kind of ugly but this is how it could potentially look:
![Screen Shot 2019-10-11 at 10 09 10 AM](https://user-images.githubusercontent.com/29791650/66658384-5bc7dc00-ec0f-11e9-9715-8f37de490796.png)

**Comment by cowboyd** [comment]
We can leave both, but be very detailed about what each one means:

e.g.

A preview package of this pull request has been released to NPM with the tag `mk_test_yeah`. You can try it out by running the following command:

```
$ npm install @minkimcello/georgia@mk_test_yeah
```

or updating your package.json to:

```json
{
  "@minkimcello/georgia": "mk_test_yeah"
}
```

Once the branch associated with this tag is deleted (usually once the PR is merged or closed), it will no longer be available. However, it currently references [`@minkimcello/georgia@1.0.12-aodfddasdf`](https://www.npmjs.com/package/@minkimcello/georgia/v/1.10.12-aodfddasdfi2d2ssj-5ccbb92) which will be available to install forever.

**Comment by minkimcello** [comment]
How's this?

![Screen Shot 2019-10-11 at 6 10 30 PM](https://user-images.githubusercontent.com/29791650/66687897-73748400-ec52-11e9-8684-37cb33be9882.png)

**Comment by cowboyd** [comment]
Fantastic.

**Comment by cowboyd** [comment]
Addressed by #21
